Python/PRP/GUI/pyGUIControlHandlers.h

```diff
@@ -26,4 +26,28 @@ PY_WRAP_PLASMA(GUICloseDlgProc, class pfGUICloseDlgProc);
 PY_WRAP_PLASMA(GUIConsoleCmdProc, class pfGUIConsoleCmdProc);
 PY_WRAP_PLASMA(GUIPythonScriptProc, class pfGUIPythonScriptProc);
 
+PyObject* ICreateGUIControlHandler(class pfGUICtrlProcObject*);
+
+/* Python property helpers */
+#define PY_PROPERTY_GUIPROC_READ(myType, name, getter)                  \
+    PY_GETSET_GETTER_DECL(myType, name) {                               \
+        return ICreateGUIControlHandler(self->fThis->getter());         \
+    }
+
+#define PY_PROPERTY_GUIPROC_WRITE(pyType, myType, name, setter)         \
+    PY_GETSET_SETTER_DECL(myType, name) {                               \
+        PY_PROPERTY_CHECK_NULL(name)                                    \
+        if (!py##pyType##_Check(value)) {                               \
+            PyErr_SetString(PyExc_TypeError, #name " expected type pf" #pyType); \
+            return -1;                                                  \
+        }                                                               \
+        self->fThis->setter(((py##pyType*)value)->fThis);              \
+        return 0;                                                       \
+    }
```

Comment:
I believe the handlers can be NULL...  If so, you should allow `Py_None` in the setter to set a NULL handler